modules/renter/hostdb/hostweight.go

```diff
@@ -186,7 +186,7 @@ func storageRemainingAdjustments(entry modules.HostDBEntry) float64 {
 // version reported by the host.
 func versionAdjustments(entry modules.HostDBEntry) float64 {
 	base := float64(1)
-	if build.VersionCmp(entry.Version, "1.1.2") < 0 {
+	if build.VersionCmp(entry.Version, "1.1.2") <= 0 {
```

Comment:
yes, the expectation is that we modify the code after every release, because each release potentially adds important changes which change how much the newer hosts should be prioritized.

v1.1.2 for example adds a critical bugfix, and so hosts running older  than 1.1.2 should be penalized pretty heavily.

So, this fix was definitely an incomplete fix.